### PR TITLE
Introduces Dagoma Sigma printer and enhance Pro430 profiles

### PR DESCRIPTION
**Pull request by 0r31**
Hi,

As a Dagoma collaborator, I want to : 
- add one delta printer (and its profiles -variants, qualities-) called Sigma.
- enhance profiles of the Pro430 printer.

I can guarantee that for now, no user has played with the existing Pro430 definition and profiles.

Please don't hesitate to make any useful comments if something is wrong according to your [profile creation guidelines](https://github.com/Ultimaker/Cura/wiki/Profiles-&-Settings).

Regards,
Orel

### CODE REVIEW

**Comment by 0r31** [comment]
Hi,

In the last commit, i forced inset_direction default.
I checked [the last commit](https://github.com/Ultimaker/Cura/commit/2a4c8aa3fd24325e57fa339cb5ed7eb6caf3fb1a#diff-b706ffec2ab072365540c0ae2879b445f76f6d4560a68569ea0c2a5a136e61fc) regarding this default value, but i prefer force it in definitions, just in case.

**Comment by jellespijker** [comment]
Can you sync your fork, with our `main`. We have added a **printer-linter** tool for our printer definitions and profiles. This should help us with processing our PR's. It performs some standard diagnostic checks and apply auto-formatting on the changed files in this PR, but you can also run it locally. See https://github.com/Ultimaker/Cura/tree/main/printer-linter

If it finds issues for which it knows a fix it should post a comment with that suggested fix such that you can easily commit that fix if you're okay with it.

File: resources/definitions/dagoma_pro_430_base.def.json [hunk 1]
```diff
@@ -1,195 +1,40 @@
...
+        "has_machine_quality": true,
+        "has_variants": true,
+        "variants_name": "Nozzle"
+    },
+
+    "overrides":
+    {
+        "machine_gcode_flavor": { "default_value": "Marlin" },
+        "machine_heated_bed": { "default_value": true },
+        "machine_center_is_zero": { "default_value": false },
```
[the remark this answers is not in the document]
Comment (0r31) [reply resources/definitions/dagoma_pro_430_base.def.json]: Done in https://github.com/Ultimaker/Cura/pull/13494/commits/3a38c9625e19d08da3698ff2cca0179a26fff81c

File: resources/definitions/dagoma_pro_430_bowden.def.json [hunk 2]
```diff
@@ -1,40 +1,26 @@
...
+        "preferred_variant_name": "Brass 0.4mm",
+        "preferred_quality_type": "h0.2"
+    },
+
+    "overrides":
+    {
+        "machine_name": { "default_value": "Dagoma PRO 430 Bowden" },
+        "machine_extruder_count": { "default_value": 1 },
+        "machine_height": { "default_value": 350 },
+        "nozzle_disallowed_areas": { "default_value": [ [[-215, 160], [-166, 160], [-215, 111]], [[215, 160], [166, 160], [215, 111]] ] },
```
[the remark this answers is not in the document]
Comment (0r31) [reply resources/definitions/dagoma_pro_430_bowden.def.json:20]: Done in https://github.com/Ultimaker/Cura/pull/13494/commits/3a38c9625e19d08da3698ff2cca0179a26fff81c

File: resources/extruders/dagoma_sigma_extruder.def.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "metadata": {
+        "machine": "dagoma_sigma",
+        "position": "0"
+    },
+
+    "overrides": {
+        "extruder_nr": {
+            "default_value": 0
+        },
+        "machine_nozzle_size": {
```
[the remark this answers is not in the document]
Comment (0r31) [reply resources/extruders/dagoma_sigma_extruder.def.json:14]: I let this parameter defined as most of extruders definitions do the very same (including Ultimaker ones).
The related commit reverting it back is https://github.com/Ultimaker/Cura/pull/13494/commits/f8298dc97e52efd457874ec30141e37e4e702eba

**Comment by 0r31** [comment]
Hi @jellespijker ,

I synced the PR branch with your main and committed some changes regarding the printer-linter suggestions.
Great tool by the way!

File: resources/extruders/dagoma_sigma_extruder.def.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (jellespijker) [reply resources/extruders/dagoma_sigma_extruder.def.json:14]: good call, and I agree. They're suggestions, after all, and meant to rethink if they need to explicitly override in the first place, there are valid reasons to do that

**Review by jellespijker — APPROVED** [review]
LGTM

I will merge these now. For next time (atleast I hope we will see more contributions ;-) ) Try to limit the PR's to 1 printer at a time

**Comment by jellespijker** [comment]
Thank you for your contribution, Your printers should be in our next release Cura 5.3

**Comment by 0r31** [comment]
@jellespijker 

Many thanks for your support.
For possible new contributions related to new definition(s), I will follow your advice. As far as I understand the rule is : one definition = one PR.

I want to share you a thought regarding the printer-linter tool. In the suggestions it made on this PR, one suggestion was to remove a parameter setting from a "child" definition as it was set in the parent one.

In the case of "abstract" definition, maybe the suggestion would be to remove the param setting if all children definitions set it.

Hope the description I make is clear enough, don't hesitate to ask me more details if unclear.

**Pull request merged**